### PR TITLE
1.2 [BE] - Integrate signup process

### PR DESCRIPTION
**Pull request by Dixtir**
[Asana Link]
[Task Link](https://app.asana.com/0/1203660811930047/1203755537679044)

[Commands]
cd frontend
npm run start

cd backend
npm run dev

[Pre-Conditions]
None

[Expected Output]
Must be able to register the user's data coming from the frontend.

[Notes]
None

[Screenshots]
Screenshot (147)

### CODE REVIEW

File: frontend/src/configs/axios.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import axios from "axios"
+import {DEV_BASE_URL} from "."
+
+export const api = axios.create({baseURL: DEV_BASE_URL})
```
Comment (ejanton-sun) [inline frontend/src/configs/axios.ts]: add new line

File: frontend/src/enums/endpoints.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum EEndpoints {
+  REGISTER_USER = "/create",
+  LOGIN = "/login"
+}
```
Comment (ejanton-sun) [inline frontend/src/enums/endpoints.ts]: add new line

**Review by ejanton-sun — CHANGES_REQUESTED** [review]

File: frontend/src/configs/axios.ts (hunk 1 above)
Comment (Dixtir) [reply frontend/src/configs/axios.ts]: added new commit

File: frontend/src/enums/endpoints.ts (hunk 2 above)
Comment (Dixtir) [reply frontend/src/enums/endpoints.ts]: added new commit

**Review by ejanton-sun — APPROVED** [review]

**Pull request merged**